release/Makefile

```diff
@@ -232,12 +232,12 @@ dvd: packagesystem
 		-DDB_FROM_SRC
 # Copy distfiles
 	mkdir -p ${.TARGET}/usr/freebsd-dist
-	for dist in MANIFEST $$(ls *.txz | grep -v -- '(base|lib32|lib64)-dbg'); \
+	for dist in MANIFEST $$(ls *.txz | grep -v -- '(base|lib32|lib64|lib64c)-dbg'); \
```

Comment:
`-vE` for consistency while you're here?

Reply:
That’s a change of behaviour and means the code isn’t actually doing what is intended upstream. Not sure why it works then, presumably it just doesn’t matter…

Reply:
My guess is the intent is -dbg distfiles require downloading but in fact they’re inadvertently included on dvd but not disc1/memstick?

Reply:
I think it makes sense to ship them on dvd FWIW, so the upstream fix might be to just "remove dead code".  I would rather not change it here though as part of this, but keep your change as-is and let upstream figure out what it wants to do with dbg dists on dvd.

Reply:
Yeah. The NO_ROOT code below does pass -E though so there's likely a discrepancy there depending on whether you use makefs as root with a real tree or makefs as non-root with a METALOG...